app/lib/src/ui/screens/about_screen.dart

```diff
@@ -1,10 +1,12 @@
+import 'package:auto_route/auto_route.dart';
 import 'package:flutter/material.dart';
 import 'package:mondaymorning/src/services/themes/size_config.dart';
 import 'package:mondaymorning/src/ui/components/more/about/cc_card.dart';
 import 'package:mondaymorning/src/ui/components/more/about/email_card.dart';
 import 'package:mondaymorning/src/ui/components/more/about/map_card.dart';
 import 'package:mondaymorning/src/ui/components/more/about/social_card.dart';
 
+@RoutePage(name: 'AboutRoute')
```

Comment:
Maintain consistency

```suggestion
@RoutePage<void>(name: 'AboutRoute')
```